frontend/src/components/Footer/Footer.jsx

```diff
@@ -1,98 +1,71 @@
-          <p className="fr-footer__content-desc">
-            Le code source est ouvert et les contributions sont bienvenues.
-            {' '}
-            <a
-              title="Voir le code source"
-              href="https://github.com/betagouv/sante-psy"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Voir le code source
-            </a>
-          </p>
-          <ul className="fr-footer__content-list">
-            <li className="fr-footer__content-item">
-              <HashLink
-                className="fr-footer__content-link"
-                title="Contactez-nous"
-                to="/mentions-legales"
-              >
-                Contactez-nous
-              </HashLink>
-            </li>
-            <li className="fr-footer__content-item">
-              <a
-                className="fr-footer__content-link"
-                href="https://www.etudiant.gouv.fr/"
-                target="_blank"
-                rel="noreferrer"
-              >
-                etudiant.gouv.fr
-              </a>
-            </li>
-            <li className="fr-footer__content-item">
-              <a
-                className="fr-footer__content-link"
-                href="https://www.enseignementsup-recherche.gouv.fr"
-                target="_blank"
-                rel="noreferrer"
-              >
-                Enseignement supérieur
-              </a>
-            </li>
-            <li className="fr-footer__content-item">
-              <a
-                className="fr-footer__content-link"
-                href="https://beta.gouv.fr/"
-                target="_blank"
-                rel="noreferrer"
-              >
-                beta.gouv.fr
-              </a>
-            </li>
-          </ul>
-        </div>
-      </div>
-      <div className="fr-footer__bottom">
-        <ul className="fr-footer__bottom-list">
-          <FooterBottomItem
-            to="/mentions-legales#accessibilite"
-            label="Accessibilité: non conforme"
-          />
-          <FooterBottomItem
-            to="/donnees-personnelles-et-gestion-des-cookies"
-            label="Données personnelles et gestion des cookies"
-          />
-          <FooterBottomItem
-            to="/mentions-legales"
-            label="Mentions légales"
-          />
-          <FooterBottomItem
-            to="/faq"
-            label="En savoir plus"
-          />
-          <FooterBottomItem
-            to="/stats"
-            label="Nos statistiques"
-          />
-        </ul>
-      </div>
-    </div>
-  </footer>
+  <DSFooter>
+    <FooterBody description={<FooterDescription />}>
+      <Logo href="https://www.enseignementsup-recherche.gouv.fr/">
+        Ministère de l&lsquo;Enseignement Supérieur, de la Recherche et de l&lsquo;Innovation
+      </Logo>
+      { footerBodyLinks.map(item => (
+        <FooterBodyItem key={footerBodyLinks.indexOf(item)}>
```

Comment:
don't use index as keys 
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
and if you do, you can use 
.map((item, index) => {})